base/files/file_proxy_unittest.cc

```diff
@@ -192,8 +192,10 @@ TEST_F(FileProxyTest, Close) {
   EXPECT_EQ(File::FILE_OK, error_);
   EXPECT_FALSE(proxy.IsValid());
 
+#if !defined(STARBOARD)
   // Now it should pass on all platforms.
```

Comment:
What part of this isn't implemented in Starboard? I assume we'd want to implement base::Move at least

Reply:
It's this one: https://github.com/youtube/cobalt/blob/feature/all-upstream-update/base/files/file_util_starboard.cc#L485-L490

Reply:
That seems wrong, as we seem to implement both copying and deleting files, which is functionally equivalent. I'd rather correctly implement `Move`

Reply:
For prioritizations, we may want to de-prioritize the work of implementing anything that is not on trunk. This code should be ok to go as it is for now.